tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "baseUrl": "./",
+    "baseUrl": "src",
```

Comment:
is this change also needed?
Looks redundant if relative paths are being used now... Perhaps, is it to forbid absolute imports?

Reply:
That’s a nice side effect, but it does not compile after the upgrade.

Reply:
Ok, I'm approving.